onvm/onvm_mgr/onvm_stats.c

```diff
@@ -222,7 +222,7 @@ onvm_stats_clear_nf(uint16_t id) {
 }
 
 void
-onvm_stats_add_event(const char *msg, struct onvm_nf_info *nf_info) {
+onvm_stats_add_event(const char *msg, struct onvm_nf_info *nf_info, unsigned core) {
```

Comment:
Should this be more generic? I don't like adding a `unsigned core` argument as its not always used and we might need to support other arguments later. Could we provide an arg struct, with different IDs?
```
struct event_info {
       uint16_t type; 
       const char *msg;
       void *data;
};
```

We should also provide helper functions that generate this struct. For example smth like `gen_event_info(msg)` for our simple messages, that way we still have the oneline `onvm_stats_add_event` calls, but we could also do `gen_event_info_with_core_arg(msg, core)` (might need a better name)  when we need more arguments.
And then the `onvm_stats_add_event` based on the event can parse arguments appropriately.

Reply:
Ya that's a better way of doing it, I'll work on these fixes when I finish my other work this weekend